src/main/resources/META-INF/plugin.xml

```diff
@@ -50,18 +48,18 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
-            <action id="GitMachete.OverrideForkPointOfSelectedBranchAction"
-                    class="com.virtuslab.gitmachete.frontend.actions.contextmenu.OverrideForkPointOfSelectedBranchAction"
-                    icon="MacheteIcons.OVERRIDE_FORK_POINT">
-                <override-text place="GitMacheteContextMenu"/>
-            </action>
-
             <action id="GitMachete.RebaseSelectedBranchOntoParentAction"
                     class="com.virtuslab.gitmachete.frontend.actions.contextmenu.RebaseSelectedBranchOntoParentAction"
                     icon="MacheteIcons.REBASE">
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.OverrideForkPointOfSelectedBranchAction"
```

Comment:
Unrel: moved to down to keep actions starting with `Checkout` next to each other (`Checkout` and `Checkout and Rebase onto Parent...`)